packages/discord.js/src/structures/Guild.js

```diff
@@ -1362,6 +1362,31 @@ class Guild extends AnonymousGuild {
     return this.edit({ features });
   }
 
+  /**
+   * Ban up to 200 users from a guild, and optionally delete previous messages sent by the banned users. Requires
+   * both the `BAN_MEMBERS` and `MANAGE_GUILD` permissions.
+   * @param {BulkBanOptions} options The options for bulk banning users
+   * @returns {Promise<BulkBanResult>} Returns an object with `bannedUsers` key containing the IDs of the banned users
+   * and the key `failedUsers` with the IDs that could not be banned or were already banned.
+   * @example
+   * // Bulk ban users
+   * guild.bulkBan({ users: ['84484653687267328'], deleteMessageSeconds: 604_800 })
+   *   .catch(console.error);
+   */
+  async bulkBan(options) {
+    if (typeof options !== 'object') throw new DiscordjsTypeError(ErrorCodes.InvalidType, 'options', 'object', true);
+    if (!Object.hasOwn(options, 'users')) throw new Error('Object "options" has no "users" attribute');
+    const { users } = options;
+
+    if (Array.isArray(users) && users instanceof Collection) throw new Error('"users" is not an array');
```

Comment:
This is doing nothing...? 
'users' can't be an array and collection at the same time...